src/library-authoring/create-library/specs/LibraryCreatePage.spec.jsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import { BrowserRouter } from 'react-router-dom';
+import { withRouter, Router } from 'react-router';
+import { createMemoryHistory } from 'history';
+import { injectIntl } from '@edx/frontend-platform/i18n';
+import { LibraryCreatePage } from '../LibraryCreatePage';
+import { libraryCreateInitialState } from '../data';
+import { SUBMISSION_STATUS } from '../../common';
+import { ctxMount } from '../../common/specs/helpers';
+import { ROUTES } from '../../common';
+
+const InjectedLibraryCreatePage = injectIntl(withRouter(LibraryCreatePage));
+
+
+describe('create-library/LibraryCreatePage.jsx', () => {
+  const config = { STUDIO_BASE_URL: 'STUDIO_BASE_URL' };
+  let props;
+  let mockResetForm;
+  let mockCreateLibrary;
+  let mockFetchOrganizations;
+
+  beforeEach(() => {
+    mockResetForm = jest.fn();
```

Comment:
The same here, it is possible to define all these variables in the start of the file